config/config.go

```diff
@@ -24,6 +24,7 @@ type Config struct {
 	SearchAPIURL              string        `envconfig:"SEARCH_API_URL"`
 	SecretKey                 string        `envconfig:"SECRET_KEY"                 json:"-"`
 	SignElasticsearchRequests bool          `envconfig:"SIGN_ELASTICSEARCH_REQUESTS"`
+	HasPrivateEndpoints       bool          `envconfig:"ENABLE_PRIVATE_ENDPOINTS"`
```

Comment:
So a few things:

1) Missing `HasPrivateEndpoints` default value check in `config_test.go`
2) Missing `ENABLE_PRIVATE_ENDPOINTS` documented in README.md
3) So I tend to keep all variables in structs in alphabetical order, so it is easier to find them later (unless there is a more logical order to lay them out).